app/src/main/java/org/oppia/android/app/databinding/TextViewBindingAdapters.java

```diff
@@ -107,24 +106,11 @@ private static String getPluralString(
       @PluralsRes int pluralsResId,
       int count
   ) {
-    // TODO(#3841): Combine these strings together.
-    return resourceHandler.getStringInLocaleWithWrapping(
-        R.string.time_ago,
-        resourceHandler.getQuantityStringInLocaleWithWrapping(
-            pluralsResId, count, String.valueOf(count)
-        )
+    return resourceHandler.getQuantityStringInLocaleWithWrapping(
```

Comment:
Isn't this change going to remove the time_ago text altogether? I'm not sure that's what we want without the code that combines plurals. Is there a reason that was removed in this new PR?

Reply:
I figured yesterday there's this problem from CI tests. I'll work to resolve this.